app-libs/stf/src/stf_sgx.rs

```diff
@@ -483,60 +218,24 @@ pub fn shards_key_hash() -> Vec<u8> {
 	vec![]
 }
 
-pub fn is_root(account: &AccountId) -> bool {
```

Comment:
Moved this function to trusted_call.rs. Should not be a public accessible function in the Stf, as it does not have a `execute_with` wrapper.